samples_project/Assets/SampleViewer/Samples/VRSample/Scripts/ContinuousMovement.cs

```diff
@@ -1,25 +1,59 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.XR;
 using UnityEngine.XR.Interaction.Toolkit;
 using Unity.XR.CoreUtils;
+using UnityEngine.InputSystem;
+using UnityEngine.UI;
+using CommonUsages = UnityEngine.XR.CommonUsages;
 
 [RequireComponent(typeof(CharacterController))]
 public class ContinuousMovement : MonoBehaviour
 {
     public XRNode leftInputSource;
     public XRNode rightInputSource;
+    public InputAction LeftMenuAction;
+    public InputAction RightMenuAction;
+    public GameObject UICanvas;
+    public GameObject Instructions;
```

Comment:
Since we should not be accessing any of these variables in other scripts, it would be good practice to keep them private but with a SerializeField